src/www/app/outline_server_repository/outline_server_access_key.ts

```diff
@@ -0,0 +1,60 @@
+// Copyright 2018 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {makeConfig, SHADOWSOCKS_URI, SIP002_URI} from 'ShadowsocksConfig';
+
+import * as errors from '../../model/errors';
+
+import {ShadowsocksConfig} from '../config';
```

Comment:
I have some ideas on how to clean up this duplication, but that's for another discussion.

Reply:
Yeah, that will be relevant to my next PR, the one that creates the `ServerConfig`

Reply:
So basically remove our ShadowsocksConfig in favor of using the ShadowsocksConfig npm module directly?

As I understand [SIP002 is just a uri scheme](https://github.com/shadowsocks/shadowsocks-org/wiki/SIP002-URI-Scheme)

Reply:
there's also this https://shadowsocks.org/guide/sip008.html

Reply:
I'm aware of that, but there are many issues with the design and it doesn't meet our needs. It was created with a different purpose in mind.

Reply:
Okay great, so we're not following this